manifests/v1beta1/katib-controller/rbac.yaml

```diff
@@ -54,10 +54,13 @@ rules:
     resources:
```

Comment:
I think we should also update v1alpha3 RBAC also: https://github.com/kubeflow/katib/blob/master/manifests/v1alpha3/katib-controller/rbac.yaml

Reply:
Done